sources/web/datalab/polymer/modules/ApiManager.ts

```diff
@@ -216,6 +227,13 @@ class ApiManager {
   }
 
   /**
+   * Gets the user settings JSON from the server.
+   */
+  static getUserSettings() {
+    return ApiManager._xhrAsync('/_settings');
```

Comment:
What I'm trying to say is, it depends on the consumption of the settings. For datalab-files, it always needs a fresh copy of the settings, so it doesn't make sense for it to use a cached version if one exists.
When the need rises for caching settings for reuse, we can add that.